### ISSUE TITLE
OperationsMap only one Method

### ISSUE THREAD

**Issue by hendisantika** [fjbecerra/springboot-hazelcast-mongodb#2]
Hi Again,

Why in OperationsMap Class only has one metho (getMap)

What if we want to use load, loadAll, Store, storeAll etc method to store / read Collections from or to mongoDB?

Thanks again.

**Comment by fjbecerra** [comment]
yes, sure, you can add those methods. This project is just for demo purpose.
That OperationMap is a wrap of hz Map, you can play with the available methods, the hz documentation is a great way to kick off with hz
